components/pipeline-service/staging/stone-stg-rh01/deploy.yaml

```diff
@@ -2462,10 +2462,10 @@ spec:
         settings:
           application-name: Konflux Staging
           custom-console-name: Konflux Staging
-          custom-console-url: https://konflux-ui.apps.stone-stg-rh01.l2vh.p1.openshiftapps.com/
-          custom-console-url-pr-details: https://konflux-ui.apps.stone-stg-rh01.l2vh.p1.openshiftapps.com/ns/{{
+          custom-console-url: https://console.dev.redhat.com/application-pipeline
+          custom-console-url-pr-details: https://console.dev.redhat.com/application-pipeline/ns/{{
```

Comment:
why on stage-p01 we are going to konflux-ui URL and here we go the oppisite way?

Reply:
fixed in a29f9f56889fea91f1ef10657323f1447b7d5cf4